src/io/socket.ts

```diff
@@ -66,6 +74,8 @@ export class Socket {
   /** `true` if V!00Pls protocol shall be used, `false` otherwise.  */
   private useV100Plus = true;
 
+  private _v100MessageBuffer: Buffer = Buffer.alloc(0);
```

Comment:
Can you add a line about what this is exactly ("Accumulation buffer for fragmented V100 messages"). (yes, I'm docu-nazi , but makes it soooo much easier if you look at this code again in 10 years, someone else was to do it :D )